region/maridia/inner-green/East Sand Hall.json

```diff
@@ -1591,8 +1658,39 @@
         "Perform a 1-tap to gain blue speed with a significant amount of momentum.",
         "Run through the door, and jump any time after the transition, crossing the room."
       ]
+    },
+    {
+      "id": 13,
+      "name": "Suitless Bootless Evir Freeze (Left to Center)",
```

Comment:
expert

Reply:
Springball jumping with the right timing of entering a sandfall is considered Extreme still.  You can freeze the evir at the right height and do a stationarylateralmidairmorph, but the timing isn't friendly.  I'm not looking to put bootless sandhalls into Expert, but this Evir freeze really isn't hard.

The Evir climb feels closer to something that can go into expert, but still isn't that appripriate plus it is a patience strat.